packages/config/config/devices/0x0129/ntm625.json

```
@@ -322,37 +321,57 @@
 			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
-			"allowManualEntry": true
-		},
-		"23": {
-			"label": "Lock Body Alarms Mask",
-			"description": "Each bit controls one alarm. 1 = alarm enabled, 0 = alarm disabled. 0xFF = All Alarms on, Default is 0xFF. Bit 0: DPS Alarm (AX) (Duplicate of parameter 19. Values should be tied together) Bit 1: Deadbolt Alarm Bit 2: Lever Rotated (Rx) Bit 3 – 7: Not Used",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 255,
-			"defaultValue": 255,
-			"unsigned": true,
-			"readOnly": false,
-			"writeOnly": false,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "DPS Alarm (AX) (Duplicate of parameter 19. Values should be tied together)",
+					"label": "Off",
 					"value": 0
 				},
 				{
-					"label": "Deadbolt Alarm",
-					"value": 1
-				},
-				{
-					"label": "Lever Rotated",
-					"value": 2
-				},
-				{
-					"label": "Off",
+					"label": "On",
 					"value": 255
 				}
 			]
+		},
+		"23[0x00]": {
+			"label": "DPS Alarm (AX) (Duplicate of parameter 19. Values should be tied together)",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
```

If the default is 0xff then these are all 1 by default
```suggestion
			"defaultValue": 1,
```